### ISSUE TITLE
Symbol.iterator property of Array broken on Chrome 49 with polyfill, but works fine without polyfill

### ISSUE THREAD

**Issue by neelance** [Financial-Times/polyfill-library#49]
We have users on Chome 49, since it is the last Chrome that works on Windows XP and Vista. On Chrome 49, reading the `Symbol.iterator` property of an `Array` gives `undefined` if the polyfill is active. For example `[][Symbol.iterator]()` gives an error. Without the polyfill it works fine.

**Comment by azizhk** [comment]
Hi, I had the same error with Chrome 43 on Android. (I think Samsung Galaxy Tab A)
Along with `default` alias I also added `Array.prototype.@@iterator` and that fixed my problem.

Though I also tested this on [BrowserStack](https://www.browserstack.com/) on Chrome 38 (this version is available in the free trial) and it seems that `[][Symbol.iterator]()` returns the iterator without polyfill. So I think `Symbol.iterator` polyfill might be broken in those versions.
